just-premium/just-premium-htb-validator.js

```diff
@@ -10,7 +10,6 @@
  * prior written permission of Index Exchange.
  */
 
-
```

Comment:
Hi!

Thanks for fixing the lint but can you please remove the changes done on the just-premium files?
Just leave the changes done for `colossus/CHANGES.md` and `colossus/colossus-htb.js`. 

Thanks